execute/exectypes/costly_messages.go

```diff
@@ -15,6 +15,12 @@ import (
 	cciptypes "github.com/smartcontractkit/chainlink-ccip/pkg/types/ccipocr3"
 )
 
+const (
+	EvmWordBytes             = 32
+	ConstantMessagePartBytes = 10 * 32 // A message consists of 10 abi encoded fields 32B each (after encoding)
```

Comment:
Internal.sol marked the overhead as 14 lots, see
```
ANY_2_EVM_MESSAGE_FIXED_BYTES = 32 * 14;
```
it's 14 as opposed to 10 because EVM ABI encoding adds 2 slots (offset + length) for every dynamic struct field